### DISCUSSION TITLE
Semantic Annotations for baseline ddppo model

### DISCUSSION THREAD

**Discussion by Claessens14** [facebookresearch/habitat-lab#1217] [Q&A]
Hi,
I am trying to run the ddppo model with semantic annotations. I using the rgbs agent in `habitat-lab/habitat/config/habitat/simulator/agents/` and added sensors to `habitat-lab/habitat/config/benchmark/nav/objectnav/`.  code screenshots attached below. I am rather new to hydra configuration tooling. Using this setup, the program runs, however the observations do not include the semantic sensor. I included what observations I currently see. 
This is my run time command: `python habitat-baselines/habitat_baselines/run.py --config-name=objectnav/ddppo_objectnav_v2_hm3d_stretch_semantic.yaml `

<img width="844" alt="Screenshot 2023-03-23 at 9 13 25 PM" src="https://user-images.githubusercontent.com/18176078/227399843-cc9222f8-c606-4e53-a616-28a0f80db794.png">

<img width="699" alt="Screenshot 2023-03-23 at 9 14 39 PM" src="https://user-images.githubusercontent.com/18176078/227399845-5dd60415-29f0-471a-923f-e57a2307a0b7.png">

<img width="512" alt="Screenshot 2023-03-23 at 9 22 04 PM" src="https://user-images.githubusercontent.com/18176078/227400374-41a6ae32-deeb-43d7-af50-09ede524b7c8.png">

**Comment by Jaraxxus-Me** [comment]
Hi, have you resolved this problem? I'm also facing the same problem when getting semantic annotations from the Env.

**Reply by Claessens14** [reply]
I recall I got it working by using an older version of habitat. specifically the habitat-challenge-2022 branch. I have tried again with the latest version.

**Comment by edwardjjj** [comment]
> Hi, have you resolved this problem? I'm also facing the same problem when getting semantic annotations from the Env.

Here is the problem about semantic annotations I'm facing #1836. Check it out to see if it solves your problem.
